coffee_core/src/coffee.rs

```diff
@@ -548,6 +548,50 @@ impl PluginManager for CoffeeManager {
         }
         Ok(nurse_actions)
     }
+
+    async fn tip(
+        &mut self,
+        plugins: &[&str],
+        amount_msat: u64,
+    ) -> Result<Vec<CoffeeTip>, CoffeeError> {
+        let plugins = self
+            .config
+            .plugins
+            .iter()
+            .filter(|plugin| plugins.contains(&plugin.name().as_str()))
+            .collect::<Vec<_>>();
+        let mut tips = Vec::new();
+        for plugin in plugins {
+            let Some(tipping) = plugin.tipping_info() else {
+                continue;
+            };
+            // FIXME write a tip_plugin method as method
+            #[derive(Debug, Deserialize)]
+            struct FetchResult {
+                invoice: String,
+            }
+            let invoice: FetchResult = self
+                .cln(
+                    "fetchinvoice",
+                    json!({
+                        "offer": tipping.bolt12,
+                        "amount_msat": amount_msat,
+                    }),
+                )
+                .await?;
+            let tip: CoffeeTip = self
+                .cln(
+                    "pay",
+                    json!({
+                        "bolt11": invoice.invoice,
```

Comment:
The is paying "bolt11" but the field in the manifest is "bolt12"

Reply:
only a bolt11 can be paid, at the previous line we use the bolt12 to fetch a bolt11